_posts/2023-6-20-credential-suppression.markdown

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
```

Comment:
We have a new layout `postv2` which is a bit more modern, shouldn't effect content, just visual